docs/trulens_eval/getting_started/core_concepts/index.md

```diff
@@ -1,5 +1,90 @@
-# Core Concepts
+# ⭐ Core Concepts
 
-This is a section heading page. It is presently unused. We can add summaries of
-the content in this section here then uncomment out the appropriate line in
-`mkdocs.yml` to include this section summary in the navigation bar.
+- ☔ [Feedback Functions](feedback_functions.md).
+
+- ⟁ [Rag Triad](rag_triad.md).
+
+- 🏆 [Honest, Harmless, Helpful Evals](honest_harmless_helpful_evals.md).
+
+## Glossary
+
+General and 🦑_TruLens-Eval_-specific concepts.
+
+- `Agent`. A `Component` of an `Application` that performs some related set of
+  tasks potentially interfacing with some external services or APIs.
+
+- `Application` or `App`. An "application" that is tracked by 🦑_TruLens-Eval_.
+  Abstract definition of this tracking corresponds to
+  [App][trulens_eval.app.App]. We offer special support for _LangChain_ via
+  [TruChain][trulens_eval.tru_chain.TruChain], _LlamaIndex_ via
+  [TruLlama][trulens_eval.tru_llama.TruLlama], and _NeMo Guardrails_ via
+  [TruRails][trulens_eval.tru_rails.TruRails] `Applications` as well as custom
+  apps via [TruBasicApp][trulens_eval.tru_basic_app.TruBasicApp] or
+  [TruCustomApp][trulens_eval.tru_custom_app.TruCustomApp], and apps that
+  already come with `Trace`s via
+  [TruVirtual][trulens_eval.tru_virtual.TruVirtual].
+
+- `Chain`. A _LangChain_ `App`.
+
+- `Completion`, `Generation`. The process or result of LLM responding to some
+  `Prompt`.
+
+- `Component`. Part of an `Application`.
+
+- `Embedding`. A real vector representation of some piece of text. Can be used
+  to find related pieces of text in a `Retrieval`.
+
+- `Eval`, `Evals`, `Evaluation`. Process or result of method that scores the
+  outputs or aspects of a `Trace`. In 🦑_TruLens-Eval_, our scores are real
+  numbers between 0 and 1.
+
+- `Feedback`. See `Evaluation`.
+
+- `Feedback Function`. A method that implements an `Evaluation`. This
+  corresponds to [Feedback][trulens_eval.feedback.feedback.Feedback].
+
+- `Generation`. See `Completion`.
+
+- `Human Feedback`. A feedback that is provided by a human, e.g. a thumbs
+  up/down in response to a `Completion`.
+
+- `LLM`, `Large Language Model`. The `Component` of an `Application` that
```

Comment:
Skip this definition? LLMs do a lot (even in our sense) more than completion

Reply:
Do you mean in non-strictly-technical definition of the term? The technical definition comes from "language model" which is exactly a next token prediction model, i.e. a model that completes a piece of text.

Reply:
(What is the non-strictly-technical part of the definition that it is missing?)

Reply:
Yeah, in the non-strictly-technical sense, it's used for *eval*, tool selection, routing, etc. Get your point though, that these are still completion tasks